tests/src/test/scala/com/scylladb/migrator/alternator/MigratorSuite.scala

```diff
@@ -112,18 +112,44 @@ trait MigratorSuite extends munit.FunSuite {
     checkSchemaWasMigrated(
       tableName,
       sourceTableDesc.keySchema,
-      sourceTableDesc.attributeDefinitions
+      sourceTableDesc.attributeDefinitions,
+      sourceTableDesc.localSecondaryIndexes,
+      sourceTableDesc.globalSecondaryIndexes
     )
   }
 
   /** Check that the table schema in the target database is equal to the provided schema */
-  def checkSchemaWasMigrated(tableName: String, keySchema: java.util.List[KeySchemaElement], attributeDefinitions: java.util.List[AttributeDefinition]): Unit = {
+  def checkSchemaWasMigrated(
+                              tableName: String,
+                              keySchema: java.util.List[KeySchemaElement],
+                              attributeDefinitions: java.util.List[AttributeDefinition],
+                              localSecondaryIndexes: java.util.List[LocalSecondaryIndexDescription],
+                              globalSecondaryIndexes: java.util.List[GlobalSecondaryIndexDescription]): Unit = {
     targetAlternator
       .describeTable(describeTableRequest(tableName))
       .table
       .tap { targetTableDesc =>
+        // Partition key
         assertEquals(targetTableDesc.keySchema, keySchema)
-        assertEquals(targetTableDesc.attributeDefinitions, attributeDefinitions)
+
+        // Attribute definitions: commented out because of https://github.com/scylladb/scylladb/issues/19716
+        // assertEquals(targetTableDesc.attributeDefinitions, attributeDefinitions)
+
+        // Local secondary indexes: do not compare their ARN, which always unique
+        def localIndexRelevantProperties(index: LocalSecondaryIndexDescription) =
+          (index.indexName, index.keySchema, index.projection)
+        assertEquals(
+          targetTableDesc.localSecondaryIndexes.asScala.map(localIndexRelevantProperties),
+          localSecondaryIndexes.asScala.map(localIndexRelevantProperties)
+        )
+
+        // Global secondary indexes: do not compare ARN and provisioned throughput (see https://github.com/scylladb/scylladb/issues/19718)
```

Comment:
Note that both scylladb/scylladb#19716 and scylladb/scylladb#19718 are not real blocker, they only make our tests a bit weaker than expected.